src/krate/mod.rs

```diff
@@ -366,10 +364,16 @@ impl Crate {
             repository,
             ..
         } = self;
-        let versions_link = match versions {
-            Some(..) => None,
-            None => Some(format!("/api/v1/crates/{}/versions", name)),
-        };
```

Comment:
Was this condition really necessary? (It removes the link if any versions are passed to `encodable`.